arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -0,0 +1,261 @@
+//
+// Copyright © 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class EditAndSyncFeaturesViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Initialize map with a basemap.
+            let tileCache = AGSTileCache(name: "SanFrancisco")
+            let tiledLayer = AGSArcGISTiledLayer(tileCache: tileCache)
+            let map = AGSMap(basemap: AGSBasemap(baseLayer: tiledLayer))
+            // Assign the map to the map view.
+            mapView.map = map
+            
+            // Create a geodatabase sync task using the feature service URL.
+            let featureServiceString = "https://sampleserver6.arcgisonline" + ".com/arcgis/rest/services/Sync/WildfireSync/FeatureServer"
```

Comment:
@vquach2404 wonder why it's concatenation of two strings here? 

I was positive I left a comment on this last week, but can't find it now. Wonder if not adding it as part of a review had to do anything with it.

Reply:
Looking back now, I'm not sure. I may have just done that because this sample was modeled after Java and that's how they also implemented it. I will remove the concatenation if it doesn't cause any errors.